ice_in

```diff
@@ -1,450 +1,156 @@
+  debug_forcing = .true.
+  debug_model = .true.
+  diagfreq = 960
+  dumpfreq = "y"
+  dump_last = .true.
+  histfreq = "d", "m", "x", "x", "x"
   ice_ic = "./input/iced.1900-01-01-10800.nc"
-  latpnt = 90.0, -65.0
   lcdf64 = .true.
-  lonpnt = 0.0, -45.0
-  ndtd = 1
-  numax = 99
-  numin = 11
-  pointer_file = "./rpointer.ice"
-  print_global = .true.
-  print_points = .false.
-  restart_ext = .false.
-  restart_file = ""
-  restart_format = "default"
-  use_leap_years = .false.
-  version_name = "undefined_version_name"
-  write_ic = .false.
-  year_init = 1
+  npt = 35040
+  pointer_file      = './rpointer.ice'
+  print_global      = .false.
 /
 &grid_nml
   bathymetry_file = "./input/topog.nc"
-  bathymetry_format = "default"
-  close_boundaries = .false.
-  dxrect = 30.0e5
-  dyrect = 30.0e5
   grid_atm = "A"
   grid_file = "./input/grid.nc"
   grid_format = "nc"
   grid_ice = "B"
   grid_ocn = "A"
   grid_type = "tripole"
-  gridcpl_file = "unknown_gridcpl_file"
   kcatbound = 0
   kmt_file = "./input/kmt.nc"
-  nblyr = 3
+  nblyr = 1
   ncat = 5
   nfsd = 1
   nilyr = 4
   nslyr = 1
-  orca_halogrid = .false.
-  use_bathymetry = .false.
 /
 &tracer_nml
-  n_aero = 3
-  n_algae = 0
-  n_dic = 0
-  n_doc = 0
-  n_don = 0
-  n_fed = 0
-  n_fep = 0
-  n_iso = 3
-  n_zaero = 0
-  restart_aero = .false.
-  restart_age = .false.
-  restart_fsd = .false.
-  restart_fy = .false.
-  restart_iso = .false.
-  restart_lvl = .false.
-  restart_pond_lvl = .false.
-  restart_pond_topo = .false.
-  restart_snow = .false.
-  tr_aero = .false.
-  tr_fsd = .false.
-  tr_fy = .true.
-  tr_iage = .true.
-  tr_iso = .false.
-  tr_lvl = .true.
-  tr_pond_lvl = .true.
-  tr_pond_topo = .false.
-  tr_snow = .false.
 /
 &thermo_nml
-  a_rapid_mode = 0.5e-03
-  aspect_rapid_mode = 1.0
-  conduct = "MU71"
-  dsdt_slow_mode = -1.5e-07
-  kitd = 1
-  ktherm = 2
-  phi_c_slow_mode = 0.05
-  phi_i_mushy = 0.85
-  rac_rapid_mode = 10
+  dsdt_slow_mode = -5e-08
 /
 &dynamics_nml
-  advection = "remap"
-  alphab = 20.0d0
-  arlx = 300.
-  brlx = 300.
-  cf = 17.0
-  coriolis = "latitude"
-  cstar = 20.0d0
-  e_plasticpot = 2.0
-  e_yieldcurve = 2.0
   k1 = 8.0d0
-  k2 = 15.0d0
-  kdyn = 1
-  krdg_partic = 1
-  krdg_redist = 1
-  kridge = 1
-  kstrength = 1
-  ktens = 0.0
-  ktransport = 1
-  mu_rdg = 4.0
-  ndte = 120
-  pstar = 2.75d4
-  revised_evp = .false.
-  seabed_stress = .false.
-  seabed_stress_method = "LKD"
-  ssh_stress = "coupled"
-  threshold_hw = 30.0d0
 /
 &shortwave_nml
-  ahmax = 0.3
-  albedo_type = "default"
-  albicei = 0.45
-  albicev = 0.75
-  albsnowi = 0.73
+  ahmax = 0.1
+  albicei = 0.44
+  albicev = 0.86
+  albsnowi = 0.7
   albsnowv = 0.98
-  dt_mlt = 1.50
+  dt_mlt = 1.0
   kalg = 0.0
-  r_ice = 0.0
-  r_pnd = 0.0
-  r_snw = 1.25
-  rsnw_mlt = 1500.
-  shortwave = "dEdd"
+  r_snw = 0.0
   sw_dtemp = 0.01d0
   sw_frac = 1.0d0
   sw_redist = .true.
 /
 &ponds_nml
   dpscale = 1.0e-3
-  frzpnd = "cesm"
-  hp1 = 0.01
-  hs0 = 0.0
```

Comment:
retain, as this isn't the default
```
  hs0 = 0.0
```

Reply:
This is only used if shortwave=dEdd (but we used shortwave default = ccsm3)